quirks/websites-that-append-2fa-to-password.json

```diff
@@ -1,5 +1,6 @@
 [
     "etrade.com",
     "mailbox.org",
-    "usaa.com"
+    "usaa.com",
+    "auth.redhat.com"
```

Comment:
...that being said, this should be placed at the beginning of the array (so it's sorted alphabetically) 😄